dev/Breadcrumb/BreadcrumbItem.cpp

```diff
@@ -70,56 +93,62 @@ void BreadcrumbItem::RevokePartsListeners()
 
 void BreadcrumbItem::OnApplyTemplate()
 {
-    RevokePartsListeners();
-
     __super::OnApplyTemplate();
 
-    winrt::IControlProtected controlProtected{ *this };
+    if (m_isEllipsisDropDownItem)
+    {
+        UpdateEllipsisDropDownItemCommonVisualState(false /*useTransitions*/);
+    }
+    else
+    {
+        RevokePartsListeners();
 
-    m_breadcrumbItemButton.set(GetTemplateChildT<winrt::Button>(s_breadcrumbItemButtonPartName, controlProtected));
+        winrt::IControlProtected controlProtected{ *this };
 
-    if (const auto& breadcrumbItemButton = m_breadcrumbItemButton.get())
-    {
-        m_breadcrumbItemButtonLoadedRevoker = breadcrumbItemButton.Loaded(winrt::auto_revoke, { this, &BreadcrumbItem::OnLoadedEvent });
+        m_button.set(GetTemplateChildT<winrt::Button>(s_itemButtonPartName, controlProtected));
```

Comment:
>m_button [](start = 8, length = 8)

nit:Could be buttonbase ?

Reply:
Nope. ButtonBase.Flyout does not exist.  Used @ line 578.